psi4/src/psi4/libmints/matrix.cc

```diff
@@ -238,6 +238,29 @@ Matrix::Matrix(dpdfile2 *inFile)
     global_dpd_->file2_mat_close(inFile);
 }
 
+Matrix::Matrix(dpdbuf4 *inBuf)
+    : name_("None"), rowspi_(inBuf->params->nirreps), colspi_(inBuf->params->nirreps)
+{
+    matrix_ = NULL;
+    symmetry_ = 0; //inBuf->my_irrep;
```

Comment:
@robertodr `symmetry_ = 0` here is not correct. I'm not sure why it's commented out but `symmetry_ inBuf->my_irrep` is what we would want here.